modules/document_repository/jsx/uploadForm.js

```diff
@@ -171,6 +170,7 @@ class DocUploadForm extends React.Component {
       if (data == 'uploaded successfully') {
       swal('Upload Successful!', '', 'success');
       this.fetchData();
+      this.props.refreshPage();
```

Comment:
1. What's the point of fetching the data and then immediately refreshing the page? Fetching the data to discard it seems unnecessary in that case.
2. What happens to the "success" swal right above? How long do users have to see that message?

Reply:
1. You are right, I will remove fetchData.
2. Click the OK button, then close this message. 
![Screen Shot 2019-06-18 at 3 58 33 PM](https://user-images.githubusercontent.com/9876007/59715576-2e7b9c00-91e2-11e9-8f41-d7b333fb3510.png)